manim/mobject/graphing/coordinate_systems.py

```diff
@@ -3399,6 +3412,12 @@ def p2n(self, point: Point3D) -> complex:
         """Abbreviation for :meth:`point_to_number`."""
         return self.point_to_number(point)
 
+    def __matmul__(self, coord: float | complex):
```

Comment:
I just noticed that `ComplexPlane.number_to_point()`, unlike the rest of coordinate systems, does not accept a `Mobject` as its parameter. It should be changed, but let's do that in another PR.
```suggestion
    def __matmul__(self, coord: float | complex) -> Point3D:
```